File: examples/fizzbuzz/fizzbuzz.java
public class Main {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++)
        {
            String output = "";
            
            if (i % 3 == 0)
            {
                output += "Fizz";
            }
            if (i % 5 == 0)
            {
                output += "Buzz";
            }
            
            if (output == "")
            {
                System.out.println(i);
            }
            else
            {
                System.out.println(output);
            }
        }
    }
}

File: examples/fizzbuzz/fizzbuzz.cs
using System;

class Program
{
    static void Main()
    {
        int i = 1;
        
        while (i <= 100)
        {
            string output = "";
            
            if (i % 3 == 0)
            {
                output += "Fizz";
            }
            if (i % 5 == 0)
            {
                output += "Buzz";
            }
            
            if (output == "")
            {
                Console.WriteLine(i);
            }
            else
            {
                Console.WriteLine(output);
            }
            
            i += 1;
        }
    }
}
